gradle/publish.gradle

```diff
@@ -0,0 +1,78 @@
+apply plugin: 'com.novoda.build-properties'
+apply plugin: 'com.jfrog.bintray'
+apply plugin: 'maven-publish'
+
+buildProperties {
+    cli {
+        using(project)
+    }
+
+    bintray {
+        def bintrayCredentials = {
+            return isDryRun() ?
+                    ['bintrayOrg': 'n/a', 'bintrayRepo': 'n/a', 'bintrayUser': 'n/a', 'bintrayKey': 'n/a'] :
+                    new File("${System.getenv('BINTRAY_PROPERTIES')}")
+        }
+        using(bintrayCredentials()).or(cli)
+        description = '''This should contain the following properties:
+                        - bintrayOrg: name of the Bintray organisation to deploy the artifacts to
+                        - bintrayRepo: name of the repo of the organisation to deploy the artifacts to
+                        - bintrayUser: name of the account used to deploy the artifacts
+                        - bintrayKey: API key of the account used to deploy the artifacts'''
+                .stripIndent()
+    }
+}
+
+
+def pomConfig = {
+    licenses {
+        license {
+            name "The Apache Software License, Version 2.0"
+            url "http://www.apache.org/licenses/LICENSE-2.0.txt"
+            distribution "repo"
+        }
+    }
+    developers {
+        developer {
+            id "novoda"
+            name "Novoda GmbH"
+        }
+    }
+    scm {
+        url "https://github.com/novoda/aws-v4-signer-java"
+    }
+}
+
+bintray {
+
+    user = project.buildProperties.bintray['bintrayUser'].string
+    key = project.buildProperties.bintray['bintrayKey'].string
+
+    pkg {
+        userOrg = project.buildProperties.bintray['bintrayOrg'].string
+        repo = project.buildProperties.bintray['bintrayRepo'].string
+        name = "aws-v4-signer"
+        licenses = ['Apache-2.0']
+        vcsUrl = 'https://github.com/novoda/aws-v4-signer-java'
+    }
+}
+
+afterEvaluate {
+    project.publishing.publications.forEach { publication ->
+        publication.pom.withXml {
+            def root = asNode()
+            root.appendNode('name', project.name)
+            root.appendNode('description', 'A Kotlin-Multiplatform implementation of the AWS V4 signing algorithm for Android and iOS')
+            root.appendNode('url', 'https://github.com/novoda/aws-v4-signer-java')
+            root.children().last() + pomConfig
+        }
+    }
+}
+
+bintrayUpload.doFirst {
+    publications = project.publishing.publications
```

Comment:
are you sure this is needed?

Reply:
Well, let's say both examples I found do this, so I am assuming this is needed.
- https://proandroiddev.com/write-and-deploy-a-kotlin-library-compatible-with-android-and-ios-48415b49ebed
- https://github.com/russhwolf/multiplatform-settings/blob/master/gradle/publish.gradle